### PR TITLE
fixes #31: Delete-for-Binomial-Heap

### LINKED ISSUE #31: Add Adjust, Delete and Meld for Binomial Heap
Add Adjust, Delete and Meld methods to Binomial Heap

### PR DESCRIPTION
**Pull request by safwan-moha**

### CODE REVIEW

**Comment by theodesp** [comment]
@safwan-moha Great. Check my comments. Ta.

**Comment by safwan-moha** [comment]
> @safwan-moha Great. Check my comments. Ta.

@theodesp did you add comments?

**Comment by theodesp** [comment]
Aa sorry I forgot to submit them.

File: binomial/binomial_heap.go [hunk 1]
```diff
@@ -6,6 +6,7 @@
 package binomial
 
 import (
+	"math"
```
[the remark this answers is not in the document]
Comment (safwan-moha) [reply binomial/binomial_heap.go]: Okay. Added a the empty line

File: binomial/binomial_heap.go [hunk 2]
```diff
@@ -56,6 +57,53 @@ func (b *BinomialHeap) DeleteMin() heap.Item {
...
+		next.parent.item = temp
+		
+		next = next.parent
+	}
+	(*next).item = heap.Integer(math.Inf(-1))
+	p.DeleteMin()
+}
+
+// FindAny returns the address of item in the heap.
+func (b *BinomialHeap) FindAny(item heap.Item) **node {
```
[the remark this answers is not in the document]
Comment (safwan-moha) [reply binomial/binomial_heap.go]: 1. un-export it as `findAny`
2. return a single pointer as `*node`

Test cases passing

**Comment by theodesp** [comment]
Cool  Thank you!

**Comment by theodesp** [comment]
@all-contributors please add @safwan-moha for tests and code

**Pull request merged**